src/gui/transferlistwidget.cpp

```diff
@@ -393,6 +393,15 @@ void TransferListWidget::copySelectedNames() const
     qApp->clipboard()->setText(torrent_names.join("\n"));
 }
 
+void TransferListWidget::copySelectedHashes() const
+{
+    QStringList torrent_hashes;
```

Comment:
It's very difficult to work with code that disagrees with the coding guidelines in so many locations. Almost every PR I've created has had issues like these. I think files should be kept consistent and then the entire file should be updated to the new guidelines all at once.